cmd/pctl/get.go

```diff
@@ -0,0 +1,279 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	profilesv1 "github.com/weaveworks/profiles/api/v1alpha1"
+
+	"github.com/weaveworks/pctl/pkg/catalog"
+	"github.com/weaveworks/pctl/pkg/client"
+	"github.com/weaveworks/pctl/pkg/formatter"
+)
+
+func getCmd() *cli.Command {
+	return &cli.Command{
+		Name:  "get",
+		Usage: "get a profile",
+		UsageText: "pctl get [--output table|json <QUERY> --installed --catalog --version] \n\n" +
+			"   example: pctl get nginx",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "output",
+				Aliases:     []string{"o"},
+				DefaultText: "table",
+				Value:       "table",
+				Usage:       "Output format. json|table",
+			},
+			&cli.BoolFlag{
+				Name:    "installed",
+				Aliases: []string{"i"},
+				Usage:   "Get all installed profiles",
+			},
+			&cli.BoolFlag{
+				Name:    "catalog",
+				Aliases: []string{"c"},
+				Usage:   "Get all profiles in a catalog",
+			},
+			&cli.StringFlag{
+				Name:    "profile-version",
+				Aliases: []string{"p"},
+				Usage: "Get information about a profile \n\n" +
+					"   pctl [--kubeconfig=<kubeconfig-path>] get <CATALOG>/<PROFILE>\n\n" +
+					"   example: pctl get catalog/weaveworks-nginx --profile-version v0.1.0",
+			},
+		},
+		Action: func(c *cli.Context) error {
+			outFormat := c.String("output")
+
+			cl, err := buildK8sClient(c.String("kubeconfig"))
+			if err != nil {
+				return err
+			}
+			catalogClient, err := getCatalogClient(c)
+			if err != nil {
+				_ = cli.ShowCommandHelp(c, "get")
+				return err
+			}
+
+			if c.Args().Len() < 1 && c.Bool("installed") {
+				return getInstalledProfiles(cl, catalogClient, "", outFormat)
+			}
+
+			if c.Args().Len() < 1 && c.Bool("catalog") {
+				return getCatalogProfiles(catalogClient, "", outFormat)
+			}
+
+			// get all installed and catalog profiles
+			if c.Args().Len() < 1 {
+				err := getInstalledProfiles(cl, catalogClient, "", outFormat)
+				if err != nil {
+					return err
+				}
+				return getCatalogProfiles(catalogClient, "", outFormat)
+			} else {
```

Comment:
nit (my formattings definitely wrong)

```suggestion
			if c.Args().Len() < 1 {
			  if c.Bool("installed") {
			  	return getInstalledProfiles(cl, catalogClient, "", outFormat)
			  }

			  if c.Bool("catalog") {
			  	return getCatalogProfiles(catalogClient, "", outFormat)
			  }
			  
			  err := getInstalledProfiles(cl, catalogClient, "", outFormat)
			  if err != nil {
				  return err
			  }
			  return getCatalogProfiles(catalogClient, "", outFormat)
			} else {
```

Reply:
fixed 👍🏻